packages/openneuro-components/src/activity-slider/ActivitySlider.tsx

```diff
@@ -10,69 +10,81 @@ import 'slick-carousel/slick/slick-theme.css'
 import './slider.scss'
 
 export interface ActivitySliderProps {
-  sliderClass?: string
-  dots?: boolean
-  responsive?: object
-  slidesToShow?: number
-  slidesToScroll?: number
-  swipeToSlide?: boolean
-  slideHeader?: string
-  data: Record<string, any>
+  className?: string
+  sliderArgs: {
+    dots?: boolean
+    responsive?: object
+    slidesToShow?: number
+    slidesToScroll?: number
+    swipeToSlide?: boolean
+    slideHeader?: string
+  }
+  data: {
+    id: string
+    analytics: {
+      views: number
+    }
+    latestSnapshot: {
+      tag: string
+      description: {
+        Name: string
+      }
+      summary: {
+        modalities: string[]
+      }
+    }
+  }[]
 }
 
 export const ActivitySlider: React.FC<ActivitySliderProps> = ({
-  slideHeader,
-  sliderClass,
+  className,
   data,
-  accessibility = true,
-  draggable = true,
-  ...props
+  slideHeader,
+  sliderArgs,
 }) => {
   const datasets = data
+  console.log(data)
   return (
-    <div className={'container activity-slider' + ' ' + sliderClass}>
+    <div className={'activity-slider' + ' ' + className}>
       <h3>{slideHeader}</h3>
-      <Slider {...props}>
-        {datasets.map((item, index) => (
+      <Slider {...sliderArgs}>
+        {data.map(({ node, index }) => (
           <>
```

Comment:
Probably want to add the key here and use the node identity instead of the list index to keep updates from breaking the list.

Reply:
I added the Key but still get a unique key error, maybe because Slick is injecting extra markup around the inner mapped children?

Reply:
It should be key={node.id} to avoid the ordering issue, otherwise looks like it should work.